fixtures/lightmaxx/vega-zoom-wash.json

```diff
@@ -89,73 +89,77 @@
       "capabilities": [
         {
           "range": [0, 10],
-          "name": "open"
+          "name": "Open"
         },
         {
           "range": [11, 20],
-          "name": "red",
+          "name": "Red",
           "color": "#ff0000"
         },
         {
           "range": [21, 30],
-          "name": "green",
+          "name": "Green",
           "color": "#00ff00"
         },
         {
           "range": [31, 40],
-          "name": "blue",
+          "name": "Blue",
           "color": "#0000ff"
         },
         {
           "range": [41, 50],
-          "name": "white",
+          "name": "White",
           "color": "#ffffff"
         },
         {
           "range": [51, 60],
-          "name": "yellow",
+          "name": "Yellow (RG)",
           "color": "#ffff00"
         },
         {
           "range": [61, 70],
-          "name": "cyan",
+          "name": "Cyan (GB)",
           "color": "#00ffff"
         },
         {
           "range": [71, 80],
-          "name": "lightblue",
+          "name": "Lightblue (BW)",
           "color": "#64c8ff"
```

Comment:
Better `#8080ff` instead of `#64c8ff`? The manual only mentions `BW` and not `Lightblue`!